composer.json

```diff
@@ -1,5 +1,23 @@
 {
     "require": {
+        "php": ">=8.0",
```

Comment:
Wow, directly jumped to nc26 and php8. Nice. I only jumped to nc24 in integration_onedrive and still stuck with php74 :(